lcls_tools/superconducting/scLinac.py

```diff
@@ -32,23 +32,24 @@ def __init__(self, cavity):
         self.currentSlopePV: PV = PV(self.pvPrefix + "SLOPE")
         self.measuredSlopePV: PV = PV(self.pvPrefix + "SLOPE_NEW")
         
-        self.maxdrive_pv: str = self.pvPrefix + "DRV_MAX_REQ"
-        self.drivemax = 1 if self.cavity.cryomodule.isHarmonicLinearizer else 0.8
+        self.maxdrive_setpoint_pv: str = self.pvPrefix + "DRV_MAX_REQ"
+        self.saved_maxdrive_pv: str = self.pvPrefix + "DRV_MAX_SAVE"
+        self.drivemax = 1 if self.cavity.cryomodule.isHarmonicLinearizer else caget(self.saved_maxdrive_pv)
```

Comment:
Should we have a initial/default value for the drivemax of the 1.3s, in case there is no value in the pv for some reason? Or is that unlikely/impossible?

Reply:
Hmm. Good point.